sceptre/plan/actions.py

```diff
@@ -843,15 +844,20 @@ def _get_simplified_status(status):
                 "{0} is unknown".format(status)
             )
 
-    def _log_new_events(self):
+    def _log_new_events(self, after_datetime):
```

Comment:
Oh yeah, this is better. I think passing the parameters rather than using instance variables is WAY better here. Thanks for making things better!